error_code_report-0.1.0.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.exasol.com/error_code_report-0.1.0.json",
+  "type": "object",
+  "title": "Exasol error code report schema.",
+  "description": "This schema defines the structure of a report about error codes in an Exasol project.",
+  "additionalProperties": false,
+  "required": [
+    "projectName",
+    "projectVersion",
+    "errorsCodes"
+  ],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "Reference to the schema. Setting this helps your IDE to assist you while writing the definition.",
+      "enum": [
+        "https://schemas.exasol.com/error_code_report-0.1.0.json",
+        "./error_code_report-0.1.0.json"
+      ]
+    },
+    "projectName": {
+      "type": "string",
+      "title": "Name of the project.",
+      "description": "Typically the GitHub repository name."
+    },
+    "projectVersion": {
+      "type": "string",
+      "title": "Version tag of the reported release."
+    },
+    "errorsCodes": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/errorCode"
+      },
+      "description": "List of `errorCodes` declared in this project version."
+    }
+  },
+  "definitions": {
+    "errorCode": {
+      "type": "object",
+      "properties": {
+        "errorCode": {
+          "type": "string",
+          "pattern": "^[WFE]-[A-Z0-9]+(-[A-Z0-9]+)?-\\d$"
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message without error code. The exception message can contain placeholders like `##NAME##`. In that case you must define the placeholder in `messagePlaceholders`."
```

Comment:
Most of the time you see curly brackets as placeholders in messages. I know this could interfere with JSON. But since it's inside a string, it should be fine.